packages/app/components/Queue/StudentPopupCard.tsx

```diff
@@ -117,7 +112,8 @@ const StudentPopupCard = ({
               cancelText="No"
               onConfirm={() => {
                 onClose();
-                updateQuestion(question, ClosedQuestionStatus.Deleted);
+                updateQuestion(question, OpenQuestionStatus.TADeleted);
+                API.questions.notify(question.id);
```

Comment:
This uses the can't find message right now for the notification, should be using the `REMOVED` message instead